BundesIdent/TCA/Coordinator.swift

```diff
@@ -191,7 +202,37 @@ struct Coordinator: ReducerProtocol {
     func token(state: inout State, action: Action) -> EffectTask<Action> {
         switch action {
         case .openURL(let url):
-            return handleURL(state: &state, url)
+            if state.remoteConfiguration.finished {
+                return handleURL(state: &state, url)
+            } else {
+                state.deferredTokenURL = url
+                return .none
+            }
+        default:
+            return .none
+        }
+    }
+
+    func remoteConfigurationRouting(state: inout State, action: Action) -> EffectTask<Action> {
+        switch action {
+        case .onAppear:
```

Comment:
Not for this PR, as we use `onAppear` unsafely elsewhere: We should definitely introduce a `onInitialAppear` as `onAppear` is not guaranteed to be called only once.

I have added the ticket USEID-939 for that.